server/src/main/java/org/elasticsearch/cluster/routing/allocation/WriteLoadConstraintSettings.java

```diff
@@ -92,7 +92,7 @@ public boolean disabled() {
      */
     public static final Setting<TimeValue> WRITE_LOAD_DECIDER_QUEUE_LATENCY_THRESHOLD_SETTING = Setting.timeSetting(
         SETTING_PREFIX + "queue_latency_threshold",
-        TimeValue.timeValueSeconds(30),
+        TimeValue.timeValueSeconds(5),
```

Comment:
This seems independent of this PR?

I would prefer a separate PR changing this. My inclination would be 10s rather than 5s for the initial version, but discussing it here does not seem right?